### ISSUE TITLE
rav1e: rust.build failed

### ISSUE THREAD

**Issue by Whitehawkx** [m-ab-s/media-autobuild_suite#2573]
I've been receiving the below error for weeks. This is on a non-free 64 bit build with most features enabled. This renders me unable to build. Does anyone have any ideas what might be happening?

--------------------------------------

┌ rav1e git  .......................................... [Recently updated]
├ Running rust.update...
├ Running rust.build...
Likely error (tail of the failed operation logfile):
  E:\media-autobuild_suite\build\rav1e-git\src/x86/sse.asm:473: warning: trailing garbage after macro name ignored [-w+pp-trailing]
  E:\media-autobuild_suite\build\rav1e-git\src/x86/sse.asm:444: ... from macro `WEIGHTED_SSE' defined here
  E:\media-autobuild_suite\build\rav1e-git\src/x86/sse.asm:474: warning: trailing garbage after macro name ignored [-w+pp-trailing]
  E:\media-autobuild_suite\build\rav1e-git\src/x86/sse.asm:444: ... from macro `WEIGHTED_SSE' defined here
  thread 'main' panicked at build.rs:147:7:
  NASM build failed. Make sure you have nasm installed or disable the "asm" feature.
  You can get NASM from https://nasm.us or your system's package manager.

  error: nonzero exit status: exit code: 0xc0000005
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
rust.build failed. Check E:/media-autobuild_suite/build/rav1e-git/ab-suite.rust.build.log
This is required for other packages, so this script will exit.

--------------------------------------
[logs.zip](https://github.com/m-ab-s/media-autobuild_suite/files/13855205/logs.zip)
[ab-suite.rust.build.log](https://github.com/m-ab-s/media-autobuild_suite/files/13855213/ab-suite.rust.build.log)

**Comment by 1480c1** [comment]
have not been able to reproduce, have you tried deleting the rav1e-git folder? Other things to try is deleting the msys64/opt folder, or just removing msys64 and mintty.lnk and trying again

**Comment by Whitehawkx** [comment]
Thanks for taking a look. I've tried deleting the rav1e-git and msys64 folder, but error still happens. Also tried just rebuilding entirely to be sure but it persists. It seems like it can't find nasm for some reason.

**Comment by hydra3333** [comment]
Unable to reproduce here either, having successfully built it today.

Dumb Query: did you install rust et al globally, separate to MABS ?

Have you tried to delete the entire MABS tree and started afresh ?

**Comment by Whitehawkx** [comment]
Just wanted to confirm that I did not install rust globally or anything like that. I tried completely removing the build folder and starting over from scratch. I also did a fresh windows install and cmos reset and running the build again before reinstalling anything. In addition to this, I've tried running the build from the same ssd as my os, and from another ssd. All still fail with the same error message. As a last resort I tried building on a separate laptop and it built successfully. Appears to be something specific to my PC, but it's strange how doing a fresh windows install didn't resolve it. I can't imagine it's my specs but running out of ideas here.
